doc/Development_Documentation/02_MVC/04_Routing_and_URLs/04_Redirects.md

```diff
@@ -1,5 +1,7 @@
 # URLs based on Redirects
 
+To use this feature please enable the `PimcoreSeoBundle` in your bundles.php file.
```

Comment:
Maybe we should also mentioned that you have to install the bundle. Just enabling it, won't work right?